src/Hero/index.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import { Flex } from '@react-spectrum/layout'
+import { View } from '@react-spectrum/view'
+import '@spectrum-css/typography'
+import PropTypes from 'prop-types'
+import cloneElement from '../utils'
+
+const Hero = ({
+  background = '#1D7DEE',
+  textColor = 'var(--spectrum-global-color-gray-200)',
+  heading,
+  text,
+  image
+}) => {
+  return (
+    <section
+      css={css`
+        height: var(--spectrum-global-dimension-static-size-3400);
+        margin-bottom: var(--spectrum-global-dimension-static-size-400);
+        background-color: ${background};
+      `}
+    >
+      <Flex height='100%' alignItems='center'>
+        <View marginStart='size-800'>
+          {cloneElement(heading, {
+            className: 'spectrum-Heading--XL',
+            css: css`
+              margin-bottom: var(--spectrum-global-dimension-static-size-200);
+              color: ${textColor};
+            `
+          })}
+
+          {cloneElement(text, {
```

Comment:
We'll have to figure out a way to limit the length of this text to 160 characters as per the Xd spec.

Reply:
Mhh I would prefer content writers to be careful than us having to limit characters because showing an ellipsis or truncating a hero text looks bad...